modules/ROOT/pages/osv2-tutorial.adoc

```diff
@@ -393,66 +387,65 @@ image::osv2-set-payload.png[]
 
 === To Deploy Your Application in Anypoint Platform
 
-. In Studio, right-click your application's name in Package Explorer and click
-Anypoint Platform > Deploy to Cloud.
-. At the User login window, specify your Anypoint Platform username and password, and click Sign in. If you don't have an Anypoint Platform login, click Sign up.
-. In Runtime Manager, specify an application name. Each name is unique and becomes the URL by which your application is available in Runtime Manager. The URL has the APPLICATION_NAME.cloudhub.io format where your application resides in the cloudhub.io name space. Ensure the application name gets a green check mark for proper naming and being unique.
+. In Studio, right-click your application's name in Package Explorer and click *Anypoint Platform > Deploy to Cloud*.
+. In the user login window, specify your Anypoint Platform username and password, and click *Sign in*. If you don't have an Anypoint Platform login, click *Sign up*.
+. In Runtime Manager, specify an application name. Each name is unique and becomes the URL where your application is available in Runtime Manager. The URL has the format `APPLICATION_NAME.cloudhub.io` where your application resides in the cloudhub.io namespace. Ensure the application name has a green check mark for proper naming and being unique.
 +
-If you see this message: "You do not have permission to deploy new applications in this environment. Please select an existing application to redeploy", click the environment button in the upper left of the Runtime Manager display to change environments from a Design environment to a Production environment.
+If you see the message: *You do not have permission to deploy new applications in this environment. Please select an existing application to redeploy*, click the environment button in the upper left of the Runtime Manager display to change environments from a design environment to a production environment.
```

Comment:
Design and Production are the actual names of the environments. These should be init capped.